documentation/latest/operations/requirements.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: System Requirements
+---
+
+
+There are two kinds of requirements for project Flotta, there is the operator
+requirements (All needed to manage the devices) and some requirements that are
+needed for the device-agent.
+
+## Operator requirements:
```

Comment:
Go is missing and I think we should make explicit which Go version is required for the operator and for the device.

Reply:
No, Golang is a requirement for development, not for users. We deliver container(operator) or binary(device)